.circleci/config.yml

```diff
@@ -3,28 +3,35 @@ version: 2.1
 jobs:
   run_tests:
     docker:
-      - image: python:3.10
+      - image: node:18-bookworm
       - image: postgres
         environment:
           POSTGRES_DB: "zosia"
           POSTGRES_USER: "zosia"
           POSTGRES_PASSWORD: "zosia"
     steps:
       - checkout
+      - run:
+          name: Install python3 and pip
+          command: DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install --no-install-recommends -y python3 python3-pip
       - run:
           name: Install requirements
           command: |
-            pip3 install -r app/requirements.txt
-            pip3 install flake8 coverage
+            pip3 install --break-system-packages -r app/requirements.txt
+            pip3 install --break-system-packages flake8 coverage
```

Comment:
I think we could create a file `app/requirements.dev.txt` with these packages inside.

Reply:
Done